client/my-sites/stats/feedback/index.tsx

```diff
@@ -161,6 +157,7 @@ function FeedbackCard( { clickHandler }: FeedbackPropsInternal ) {
 function StatsFeedbackController( { siteId }: FeedbackProps ) {
 	const [ isOpen, setIsOpen ] = useState( false );
```

Comment:
No related to the PR, but I guess `isOpen` could use a better name since there are two open statuses for the child components.

Reply:
Agreed. This was named before the floating panel was merged so there was only a single view to track at that time. It definitely makes sense to update the naming now.